packages/pn-pa-webapp/src/pages/components/NewNotification/Recipient.validations.ts

```diff
@@ -3,19 +3,6 @@ import { dataRegex, RecipientType } from '@pagopa-pn/pn-commons';
 import { NewNotificationRecipient, PaymentModel } from '../../../models/NewNotification';
 import { getDuplicateValuesByKeys } from '../../../utils/notification.utility';
 
-export function denominationTotalLength(
```

Comment:
If I remember well, it was me who created `denominationTotalLength` function, along with differentiated messages about total denomination length for PF and PG. Some subsequent modification, not made by me, eliminated both the use of the function and the differentiated messages, so that the i18n entries are not anymore in `notifiche.json`, only this function remained but it had no uses. So I made the obvious thing, i.e. delete it. 
The logic about the denomination' length validation is now implemented in `Recipient.tsx`, again, it was not me who decided this. In this PR I just adjusted a tiny bug on that validation.  
I would prefer to avoid the back-and-forth modifications, hence I would leave the code in its present form.